tidb-cloud/tidb-cloud-auditing.md

```diff
@@ -138,22 +145,22 @@ For more information, see [Creating storage buckets](https://cloud.google.com/st
 
 #### Step 3. Enable audit logging
 
-In the TiDB Cloud console, go back to the **Audit Logging** dialog box where you got the TiDB Cloud account ID, and then take the following steps:
+In the TiDB Cloud console, go back to the **Enable Database Audit Logging** dialog box where you got the TiDB Cloud account ID, and then take the following steps:
 
 1. In the **Bucket URI** field, enter your full GCS bucket name.
```

Comment:
```suggestion
1. In the **Bucket URL** field, enter your full GCS bucket name.
```

Reply:
This is a wrong modification of the Staging environment. The original name "URI" is retained in the release. So keep "URI" in the document.